system/Commands/Encryption/GenerateKey.php

```diff
@@ -212,6 +212,6 @@ protected function keyPattern(string $oldKey): string
             $escaped = "[$escaped]*";
         }
 
-        return "/^[#\s]*encryption.key[=\s]*{$escaped}$/m";
+        return "/^[#\\s]*encryption.key[=\\s]*{$escaped}$/m";
```

Comment:
Is that correct? Without reading the context this looks to me like it should be `\s` for "white space character".

Reply:
I've checked this too. This is correct. Since this is not inside the context of PCRE functions, `\s` does not mean anything even inside double quotes. https://3v4l.org/YVrdo